gpkit/build.py

```diff
@@ -279,7 +271,12 @@ def build(self):
 
 def build_gpkit():
     "Builds GPkit"
-    global settings  # pylint: disable=global-variable-not-assigned
+    try:
+        import gpkit
+        log("# Moving to the directory from which GPkit was imported.")
+        os.chdir(gpkit.__path__[0])
```

Comment:
I believe that the install step will remove the already installed one before building, and this passes tests and helps with #959 so I figured it was worth a try.